CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [10.12.1] - 2020-03-27
+
+### Zero-diff to previous release: YES
+
+### Restart Changes: NO except moist_internal has an additional 3 fields: NGRAUPEL, NRAIN, NSNOW
+
+Major changes are:
+
+1. Broader support for various computing environments including SLES12. Adds a new question to gcm_setup (and other setup scripts) that asks a user what type of processor to run on. It then uses the "default" number of cores per those processors to better figure out how to handle the IOserver nodes.
+2. Changes that enable MAPL2 GEOS to work with MPT at NCCS (but still not SLES12 Haswell).
```

Comment:
It actually does work on SLES 12 Haswell!

Reply:
I will change